week-1/K-functions-parameters/exercise.js

```diff
@@ -1,9 +1,9 @@
 // Complete the function so that it takes input parameters
-function multiply() {
-  // Calculate the result of the function and return it
+function multiply(a, b) {
+  return a * b;
 }
 
 // Assign the result of calling the function the variable `result`
-var result = multiply(3, 4);
+var result = multiply((a = 3), (b = 4));
```

Comment:
hi Mussie, your function definitions is good, however but "var result = multiply((a = 3), (b = 4));" should not work.
instead you need to write "var result = multiply(3, 4); console.log(result);"